graf2d/asimage/src/TASPngWriter.cxx

```diff
@@ -0,0 +1,49 @@
+#include "TASPngWriter.h"
+
+#include <png.h>
```

Comment:
On several platforms `png.h` is not found!

This peace of code should be placed directly in `libAfterImage` - only it has proper configuration for `libpng` usage. `TASImage` is just C++ decoration around.

Reply:
What if one does not use the builtin libAfterImage? This seems to be happening with the CI builds :(